apps/SageAccountWeb/src/style/_Register.scss

```diff
@@ -1,24 +1,4 @@
 .RegisterAccount1 {
-  .back-button {
-    padding: 0;
-    width: 24px;
-    position: absolute;
-    margin: 24px;
-    top: 0px;
-    left: 0px;
-  }
-  .mainContent {
-    min-height: 350px;
-    .form-group:first-child {
-      margin-top: 50px;
-    }
-    .form-group {
-      margin-top: 30px;
-    }
-    > * {
-      width: 300px;
-    }
-  }
```

Comment:
Out of date styling, remove so that registration page logo and buttons can use full width of the left panel-- 

main | feature
:---:|:---:
![main_register1_registerGoogle](https://user-images.githubusercontent.com/26949006/230204471-2a31abee-1cd0-42f5-9e72-63cb13dbb622.png) | ![feature_register1_registerGoogle](https://user-images.githubusercontent.com/26949006/230204450-b14dee09-4ce1-488a-914e-4215b134d601.png)